librz/core/cmd.c

```diff
@@ -2087,7 +2081,7 @@ static void rz_w32_cmd_pipe(RzCore *core, char *rizin_cmd, char *shell_cmd) {
 #undef __CLOSE_DUPPED_PIPES
 #endif
 
-RZ_API int rz_core_cmd_pipe(RzCore *core, char *rizin_cmd, char *shell_cmd) {
+RZ_API int rz_core_cmd_pipe_old(RzCore *core, char *rizin_cmd, char *shell_cmd) {
```

Comment:
Could you please add Doxygen description here?

Reply:
not going to happen, since it is old code.